### PR TITLE
remove page transitions from Dashboard

### LINKED ISSUE #10124: remove transitions (page transitions)
Based on the findings in https://github.com/rancher/dashboard/pull/9998

Remove mentions/logic related to page transitions

### PR DESCRIPTION
**Pull request by aalves08**
### Summary
Fixes #10124 

### Occurred changes and/or fixed issues

- remove page transitions from Dashboard

### Technical notes summary

- seems like the package `vue2-transitions` wasn't needed anymore, so I proceeded to remove it
-

### CODE REVIEW

File: shell/components/nuxt/nuxt-child.js [hunk 1]
```diff
@@ -57,66 +43,14 @@ export default {
...
-      }
-    }
-
     let routerView = h('routerView', data)
 
     if (props.keepAlive) {
       routerView = h('keep-alive', { props: props.keepAliveProps }, [routerView])
     }
 
     return h('transition', {
```
Comment (codyrancher) [inline shell/components/nuxt/nuxt-child.js:52]: Is this still needed?

File: shell/initialize/App.js [hunk 2]
```diff
@@ -21,25 +21,10 @@ export default {
...
-    const transitionEl = h('transition', {
-      props: {
-        name: 'layout',
-        mode: 'out-in'
-      },
-      on: {
-        beforeEnter(el) {
-          // Ensure to trigger scroll event after calling scrollBehavior
-          window.$nuxt.$nextTick(() => {
-            window.$nuxt.$emit('triggerScroll');
```
Comment (codyrancher) [inline shell/initialize/App.js:33]: I think you'll still need to emit this event on page transition.

It looks like there's some transitions which won't scroll to the top of the page when switching pages.


https://github.com/rancher/dashboard/assets/55104481/3c034cb9-f914-46f8-bdff-5f209eb52af7

File: shell/components/nuxt/nuxt-child.js (hunk 1 above)
Comment (aalves08) [reply shell/components/nuxt/nuxt-child.js:52]: Not really... Can be just a plain `div` afaik.

File: shell/initialize/App.js (hunk 2 above)
Comment (aalves08) [reply shell/initialize/App.js:33]: Well, this one was driving me crazy but I just realized that it never worked! I am running a 2.8.0-rc3 instance and the UI that is built with it fails on the move from Pods to Workloads (or vice versa... one of these two still has the problem).
I haven't been able to fix it but I think it's because the Pods list view is essentially the same as Workloads, although interestingly it does trigger a route change and everything seems to be correct through my investigation in https://github.com/rancher/dashboard/blob/master/shell/utils/router.scrollBehavior.js

Comment (aalves08) [reply shell/initialize/App.js:33]: Fixed it! 💦  it wasn't registered as subtype of workloads... man, this one was tough

Comment (codyrancher) [reply shell/initialize/App.js:33]: Nice, find. Well done.

**Review by codyrancher — DISMISSED** [review]

File: shell/config/product/explorer.js [hunk 3]
```diff
@@ -103,6 +103,9 @@ export function init(store) {
     componentForType(WORKLOAD_TYPES[key], WORKLOAD);
   }
 
+  // Deal with POD (it wasn't registered as belonging to the workloads, so it wasn't using the list view of workloads)
+  componentForType(POD, WORKLOAD);
```
Comment (richard-cox) [inline shell/config/product/explorer.js:107]: Pods both are and aren't workloads. Why is this needed?

Importantly though they have their own detail and edit components which are now bypassed with this line. interestingly edit works, but detail is broken.

**Review by richard-cox — CHANGES_REQUESTED** [review]
- The extension page is broken, could be an issue with various layouts? 
  image

**Comment by aalves08** [comment]
> * The extension page is broken, could be an issue with various layouts?
>   image

Unfortunately I found this today, along with the Charts installation wizard (our e2e tests got it). I now understand why and will add a fix for it. It reverted all of broken screens to the correct state.

File: shell/config/product/explorer.js (hunk 3 above)
Comment (aalves08) [reply shell/config/product/explorer.js:107]: It was to try and fix https://github.com/rancher/dashboard/pull/10126#discussion_r1423204878 . This one was a "bug" that already existed. I prefer to revert this change and have the "same old" bug rather than breaking a complete page.

The problem is that the PODS list view currently uses a default list view and I've just noticed that if you switch between resource list INSIDE the same PRODUCT to another that uses the default list view it doesn't re-adjust the scrolling.

Yesterday I thought I understood why, but apparently I still don't 😅 

I'll revert this change.

**Review by richard-cox — APPROVED** [review]
